core/src/main/java/org/geysermc/geyser/item/type/Item.java

```diff
@@ -227,9 +237,10 @@ public void translateNbtToJava(@NonNull NbtMap bedrockTag, @NonNull DataComponen
         // TODO verify
         // TODO streamline Enchantment process
         Enchantment.JavaEnchantment enchantment = Enchantment.JavaEnchantment.of(enchantId);
-        if (enchantment == Enchantment.JavaEnchantment.SWEEPING_EDGE) {
-            addSweeping(session, builder, level);
-            return null;
+        if (ENCHANTMENT_TRANSLATION_KEYS.containsKey(enchantment)) {
```

Comment:
This can be replaced by moving the below get() above here then a null check.

Reply:
Fixed, thanks